### ISSUE TITLE
Add the Web Almanac Website!

### ISSUE THREAD

**Issue by tunetheweb** [calumryan/a11ywebsites#28]
__URL:__ https://almanac.httparchive.org

The Web Almamac is an annual “state of the web” report covering many topics as “chapters” including [Accessibility](https://almanac.httparchive.org/en/2020/accessibility). We have spent considerable time and effort to look to make the website as accessible as possible, and to act as a showcase for the many techniques discussed in the chapters.

We use well-structured, semantic, HTML with the addition of ARIA attributes were appropriate. We also use additional, newer APIs like `prefers-reduced-motion` and reducing data on limited connections or where asked by users. We aim to make the site progressively enhanced so it should still work well on the simplest HTML device. 

Being a data-driven report, we make particular efforts to ensure all data figures are well-described with both a caption and an easily-displayed description, linked with `aria-labelledby` and `aria-describedby` attributes. The source data and queries are also linked from each chapter to make the data further accessible to those wanting to analysis further.

We also make the report available in PDF format and take extra efforts to ensure it is an tagged PDF/UA-1 compliant PDF.

We perform automatic Lighthouse checks, using the underlying axe checks on [*every, single commit*](https://calendar.perfplanet.com/2020/running-lighthouse-in-github-actions/) to try to avoid regressions - particularly in Accessibility. This check also runs weekly on the project website and ensures we keep our score of 100 for *every, single* page on the website.

We also regularly test manually, using keyboard navigation and the likes of VoiceOver on Safari. Plus, we are lucky to have access to Accessibility experts involved in the report to ask questions to ([example](https://github.com/HTTPArchive/almanac.httparchive.org/issues/1769)). We give credit on the website to all contributors both in individual chapters, and on [our contributors page](https://almanac.httparchive.org/en/2020/contributors).

We are very [open and transparent on our accessibility work and issues through GitHub](https://github.com/HTTPArchive/almanac.httparchive.org/issues?q=label%3Aaccessibility+) and have so far always addressed any accessibility issues raised quickly (within days).

We aim to make the site accessible to as many people as possible and so put considerable effort into translating the content with volunteers having made some of the content available in 11 different languages. We also aim to support those languages as best as possible and have added new functionality to handle the nuances and grammar of certain languages, rather than a basic translation. For example, [the logo is different on our Japanese pages](https://almanac.httparchive.org/ja/2020/) to represent the different ordering in that language, we have added [RTL support to our templates](https://github.com/HTTPArchive/almanac.httparchive.org/pull/551) (though currently unused as that language stalled) and also support things like [Russian plurals](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1969).

We don’t limit ourselves to AA conformance and look also to AAA conformance - particularly with its enhanced colour contrasts.

I believe this site would make a good addition to your show case and would be happy to answer any questions and address and issues you spot when considering this site.

**Comment by tunetheweb** [comment]
@calumryan many thanks for reviewing the site and for identifying the reflow issues on large text and smaller viewports. I’ve just pushed a fix that I believe addresses this, but would be very keen to hear if this is not the case. Thanks again for running the site and for featuring us!

**Comment by calumryan** [comment]
That's improved for 320 pixels. Still some issues at around 1000 pixels wide as demonstrated here with the dropdown open it causes a reflow issue.

<img width="329" alt="Screenshot 2021-02-22 at 21 20 51" src="https://user-images.githubusercontent.com/2226904/108771960-43323a80-7554-11eb-9f13-c8e2e1befa4b.png">

**Comment by tunetheweb** [comment]
Thanks for looking again.

I can see what happened here: it decides the left/right hang on menu open based on whether it will overflow the screen. So can overflow if you zoom or change browser width while it's open - if you closed it and then reopened it again it would stop overflowing. Or if you'd opened it after zooming or changing size it would also not have been an issue. So a bit of an edge case, I hadn't fully considered. I added an `onresize` call to check this again and rehang it if necessary.

I also decided to change to `em`-based media queries as felt the desktop view wasn't the ideal view at those very large base font sizes except on really wide screen. So we go to mobile view more often which makes this issue less likely to happen - but still, the functionality is in there for when it does.

Both those changes are released now.
